src/app/shell/personal-cabinet/provider/create-workshop/create-workshop.component.ts

```
@@ -101,21 +153,97 @@
+      asyncScheduler.schedule(() => {
+        const stepToGo = this.getFirstInvalidStep();
+
+        if (stepToGo !== -1 && this.stepper) {
+          this.stepper.selectedIndex = stepToGo;
+        }
+      });
+    });
+  }
+
+  public isDirtyAndNotUnfinished(): boolean {
+    const isUnfinished: Boolean = this.getRouteParam() === BannerMode.Unfinished;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)